docs/ru/main/base/s3/dev/s3-golang/s3-golang-objects/s3-golang-objects.meta.json

```
@@ -0,0 +1,12 @@
+{
+  "title": "Операции с объектами",
+  "pageDescription": "",
+  "shortDescription": "Работа с объектами S3",
+  "metaTitle": "Операции с объектами - VK Cloud",
+  "metaDescription": "Проведя  предварительные настройки сессии ,  можно приступить к работе с объектами S3. Загрузка объекта Загрузка объектов в бакет инициализируется данной к",
```

Что-то совсем бессмысленное получилось, предлагаю переформулировать

```suggestion
  "metaDescription": "Доступные операции с объектами хранилища: загрузка, получение списка объектов, удаление.",
```